### PR TITLE
Google auth

### PR DESCRIPTION
**Pull request by fabianzs**

### CODE REVIEW

**Review by JohnLackland — DISMISSED** [review]
It seems good to me.

File: ASP.NET-Core-Webapp/Services/AuthService.cs [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            //    {
+            //         new Claim(ClaimTypes.NameIdentifier, sub),
+            //         new Claim(ClaimTypes.Email, email)
+            //    }),
+            //    Expires = DateTime.UtcNow.AddMinutes(5),
+
+            //    SigningCredentials = new SigningCredentials(new SymmetricSecurityKey(Encoding.ASCII.GetBytes(configuration["Authentication:Jwt:Secret"])), SecurityAlgorithms.HmacSha256Signature)
+            //};
+            //var token = tokenHandler.CreateToken(tokenDescriptor);
+            //var securetoken = tokenHandler.WriteToken(token);
```
Comment (szirmi) [inline ASP.NET-Core-Webapp/Services/AuthService.cs]: please always remove unnecessary code

File: ASP.NET-Core-Webapp/Services/AuthService.cs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        public AuthService(IConfiguration configuration)
+        {
+            this.configuration = configuration;
+        }
+
+        public string GetGoogleLogin()
+        {
+            string base_url = "https://accounts.google.com/o/oauth2/v2/auth";
+            string scope = "email+openid";
+            string redirect_uri = "http://localhost:5001/auth";
```
Comment (szirmi) [inline ASP.NET-Core-Webapp/Services/AuthService.cs]: move this in development configuration
(in this case we need to change code to be able to deploy)

File: ASP.NET-Core-Webapp/Services/AuthService.cs [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+            return $"{base_url}?scope={scope}&redirect_uri={redirect_uri}&response_type={response_type}&client_id={client_id}";
+        }
+
+        public GoogleToken GetToken(string code)
+        {
+            var dict = new List<KeyValuePair<string, string>>();
+            dict.Add(new KeyValuePair<string, string>("code", code));
+            dict.Add(new KeyValuePair<string, string>("client_id", configuration["Authentication:Google:ClientId"]));
+            dict.Add(new KeyValuePair<string, string>("client_secret", configuration["Authentication:Google:ClientSecret"]));
+            dict.Add(new KeyValuePair<string, string>("redirect_uri", "http://localhost:5001/auth"));
```
Comment (szirmi) [inline ASP.NET-Core-Webapp/Services/AuthService.cs]: instead of (redundant) string argument use the variable above

**Review by szirmi — CHANGES_REQUESTED** [review]

**Review by szirmi — APPROVED** [review]

**Pull request merged**